docs/articles/documentation/reference/command-line-interface.md

```diff
@@ -970,6 +971,24 @@ The values of the `typescript.options.lib` compiler option should be identical t
 *Related configuration file property*: [compilerOptions](configuration-file.md#compileroptions).
 *Related API method*: [runner.compilerOptions](testcafe-api/runner/compileroptions.md)
 
+### --cache
+
+If enabled, TestCafe proxy caches webpage dependencies (like styles and scripts) for webpages visited during tests. When TestCafe revisits the page, it pulls assets from the cache instead of requesting them from the server.
+
+```sh
+testcafe chrome my-tests --cache
+```
+
+TestCafe emulates the native caching mechanism of the browser. For example, in *Chrome* TestCafe only caches resources that *Chrome* itself would cache if run without TestCafe.
```

Comment:
pls don't abuse the word 'mechanism'

```suggestion
TestCafe emulates browsers' native caching behavior. For example, when working with *Chrome*, TestCafe only caches the resources that *Chrome* itself would cache in a real-life scenario.
```

Reply:
Testing with TestCafe is as `real-life` as running the browser without TestCafe.

`When working with` is an unnecessary complication. `In` should suffice

```suggestion
TestCafe emulates the browser's native caching behavior. For example, in *Chrome* TestCafe only caches resources that *Chrome* itself would cache if run without TestCafe.
```